src/main/generated/data/arsmagicalegacy/loot_tables/blocks/inscription_table.json

```diff
@@ -4,20 +4,27 @@
     {
       "bonus_rolls": 0.0,
       "conditions": [
+        {
+          "block": "arsmagicalegacy:inscription_table",
+          "condition": "minecraft:block_state_property",
+          "properties": {
+            "half": "right"
+          }
+        },
```

Comment:
previously this condition was on the entry not the table

Reply:
This was the easier way to do it in datagen. It makes no functional difference.